apps/src/client.rs

```diff
@@ -78,14 +78,24 @@ pub fn connect(
         std::net::SocketAddr::V6(_) => "[::]:0",
     };
 
+    let mut sockets =
```

Comment:
If we are only going to use at most 2 sockets in the migration, I think it would be better to make that explicit (i.e. just name the objects) rather than have a potentially unbounded vector that requires magic indexes, what do you think?

Reply:
I have no objection doing explicit naming instead of the current `sockets` solution. My original motivation to do so was to handle the variable number of sockets with slightly less code. I now have `socket` and `migrate_socket`, the second being an `Option` depending whether the application wants to perform migration or not.